server/db/models/vote.js

```diff
@@ -0,0 +1,21 @@
+const Sequelize = require('sequelize');
+const db = require('../db');
+
+module.exports = db.define('vote', {
+  upvote: {
+    type: Sequelize.BOOLEAN,
+    defaultValue: false
+  },
+  downvote: {
+    type: Sequelize.BOOLEAN,
+    defaultValue: false
+  },
```

Comment:
suggest using either an enum (e.g. `up` and `down` or else make this field `isUpvote`